zyte_common_items/components.py

```diff
@@ -39,6 +39,9 @@ class _ListMetadata(Item):
     #: the following format: ``YYYY-MM-DDThh:mm:ssZ``.
     dateDownloaded: Optional[str] = None
 
+    #: Contains paths to fields with the description of issues found with their values.
+    validationMessages: Optional[Dict[str, List[str]]] = None
```

Comment:
I think we need to updated docstrings for various metadata classes: this one, _DetailsMetadata, maybe something else.

Reply:
Yes. Maybe these docstrings should not mention their fields to begin with.

Reply:
https://github.com/zytedata/zyte-common-items/pull/88/commits/828d9da88d40a001e958d96c0c92bc2e15ab634d ?